microceph/api/endpoints.go

```diff
@@ -10,4 +10,5 @@ var Endpoints = []rest.Endpoint{
 	disksCmd,
 	resourcesCmd,
 	servicesCmd,
+	enableRGWCmd,
```

Comment:
That seems odd. I was expecting enabling services to be done through an update to the existing services endpoint.

Right now, we only have `GET /1.0/services` which gives us a pretty basic list of services and their location.

I think if I was to do this change, I'd instead change the services API to offer something like:
 - GET /1.0/services (same as we do today probably?)
 - GET /1.0/services/SERVICE (where service is osd, mon, rgw, mgr, mds) which provides us with service state and a list of locations serving it
 - PUT /1.0/services/SERVICE to alter the configuration of a service, letting you enable it and indicate where it should be enabled (setting the location)

Reply:
Thanks @stgraber , this is valuable feedback.

I have now revamped the API, adding a `PUT /1.0/services/rgw` endpoint which takes a flag to enable/disable a service. I would like to defer choosing a location for the service, as well as listing services and locations, to a separate PR if this works for you.